### PR TITLE
updated main.md

### PR DESCRIPTION
**Pull request by ayushi02**

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -93,9 +93,35 @@ It is a good practice to make a new branch for every new PR you make. Also,name
...
+  
+  3	     77b9b82	  Improving database interface	
+  
+  4	     3c9093c	  Merged develop branch into master
+  
+  5	     b3d92c5	  Adding new Event CMS Module	
+  
+  6	     7feddbb	  Adding CMS class and files	
+  
+  7	     a809379	  Adding project to Git
```
Comment (aSquare14) [inline main.md]: These many examples are not needed. 
Also write them as, 
`commit 1`
`commit 2` etc.

File: main.md [hunk 2]
```diff
@@ -93,9 +93,35 @@ It is a good practice to make a new branch for every new PR you make. Also,name
...
+  
+  6	     7feddbb	  Adding CMS class and files	
+  
+  7	     a809379	  Adding project to Git	
+  
+
+  Using the git log above we want to remove the following commit; 2 (ae45fab).
+
+  `$ git rebase --onto <branch name>~<first commit number to remove> <branch name>~<first commit to be kept> <branch name>`
+
```
Comment (aSquare14) [inline main.md]: Please add a description of git command in bullet point format below the command.

File: main.md [hunk 3]
```diff
@@ -93,9 +93,35 @@ It is a good practice to make a new branch for every new PR you make. Also,name
...
+  e.g to remove commit 2 above
+
+  `$ git rebase --onto repair~2 repair~1 repair`
```
Comment (aSquare14) [inline main.md]: Please add a description of git command in bullet point format below the command.

File: main.md (hunk 2 above)
Comment (ayushi02) [reply main.md]: added

**Comment by aSquare14** [comment]
@vibhavagarwal5 After your review, this should be good to merge. @ayushi02 Thanks for working on this :)

File: main.md [hunk 4]
```diff
@@ -92,6 +92,43 @@ It is a good practice to make a new branch for every new PR you make. Also,name
...
+   -Then give branch name along with first commit to be kept.
+   
+   -Above command could also be used to remove one or more consecutive commits.For example if you want to remove commit 2&3, command          would be as follows:
+   
+    `$ git rebase --onto repair~3 repair~1 repair`
+    
+     where first commit to be removed is 3 and first commit to be kept is 1. This would result in removing commit 2&3.
+     
+     
+=======
```
Comment (vibhavagarwal5) [inline main.md]: Remove this!

Comment (vibhavagarwal5) [reply main.md]: Can u pls remove this line.

**Comment by vibhavagarwal5** [comment]
@ayushi02 This PR is good, can u pls make the necessary changes as suggested and send. We'll merge this then!

File: main.md [hunk 5]
```diff
@@ -92,6 +92,40 @@ It is a good practice to make a new branch for every new PR you make. Also,name
...
+  
+  - This command will delete commit 2
+  
+   Generalized Format for writing command is as follows:
+   
+  `$ git rebase --onto <branch name>~<first commit number to remove> <branch name>~<first commit to be kept> <branch name>`
+  
+   -Use rebase tool to rebase a series of commits onto the HEAD they were originally based on instead of moving them to another one.
+   
+   -Then give branch name along with first commit to be removed.
```
Comment (vibhavagarwal5) [inline main.md]: add a space btw - and Use, check all the statements below and rectify those

Comment (ayushi02) [reply main.md]: okay

**Comment by vibhavagarwal5** [comment]
Looks good, merging it. Thanks for working on this!

**Pull request merged**